pkgs/applications/graphics/photivo/default.nix

```diff
@@ -10,6 +10,14 @@ stdenv.mkDerivation rec {
     sha256 = "0f6y18k7db2ci6xn664zcwm1g1k04sdv7gg1yd5jk41bndjb7z8h";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://www.linuxquestions.org/questions/attachment.php?attachmentid=17287&d=1420577220";
```

Comment:
Yeah, I agree. I have force pushed a new version that includes the patch directly. Thanks for the comment!